### PR TITLE
Update documentation for prompts

### PR DESCRIPTION
**Pull request by eyurtsev**
* Documentation to favor creation without declaring input_variables
* Cut out obvious examples, but add more description in a few places

### CODE REVIEW

File: docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx [hunk 1]
```diff
@@ -1,140 +1,122 @@
-Here's the simplest example:
+Typically, language models expect the prompt to either be a string or else a list of chat messages.
 
-```python
-from langchain import PromptTemplate
+## Prompt template
+
+`PromptTemplate` to create a template for a string prompt.
```
Comment (baskaryan) [inline docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx:5]:
```suggestion
Use `PromptTemplate` to create a template for a string prompt.
```

File: docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx [hunk 2]
```diff
@@ -1,140 +1,122 @@
...
 ## Chat prompt template
 
-[Chat Models](../models/chat) take a list of chat messages as input - this list commonly referred to as a `prompt`.
-These chat messages differ from raw string (which you would pass into a [LLM](/docs/modules/model_io/models/llms) model) in that every message is associated with a `role`.
+The prompt to [Chat Models](../models/chat) is a list of chat messages.
 
-For example, in OpenAI [Chat Completion API](https://platform.openai.com/docs/guides/chat/introduction), a chat message can be associated with the AI, human or system role. The model is supposed to follow instruction from system chat message more closely.
+Each chat message is associated with content, and an additional parameter called `role`.
 
+<<<<<<< HEAD
```
Comment (baskaryan) [inline docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx:62]: some merge conflicts

File: docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx [hunk 3]
```diff
@@ -1,140 +1,122 @@
...
-
-After that, you can build a `ChatPromptTemplate` from one or more `MessagePromptTemplates`. You can use `ChatPromptTemplate`'s `format_prompt` -- this returns a `PromptValue`, which you can convert to a string or Message object, depending on whether you want to use the formatted value as input to an llm or chat model.
-
-
-```python
-chat_prompt = ChatPromptTemplate.from_messages([system_message_prompt, human_message_prompt])
-
-# get a chat completion from the formatted messages
-chat_prompt.format_prompt(input_language="English", output_language="French", text="I love programming.").to_messages()
+from langchain.chat_models import ChatOpenAI
```
Comment (baskaryan) [inline docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx:110]:
```suggestion
from langchain.chat_models import ChatOpenAI

```

**Comment by baskaryan** [comment]
few small comments, otherwise lgtm!

File: docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx (hunk 2 above)
Comment (eyurtsev) [reply docs/snippets/modules/model_io/prompts/prompt_templates/get_started.mdx:62]: good catch

**Pull request merged**